html/Minimo/style.css

```diff
@@ -0,0 +1,258 @@
+html {
+  font-size: 2rem;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+h1, h2, h3, h4, h5, h6, p, ul {
+  margin: 0;
+}
+
+/* Font styles */
+.title--font {
+  font-family: "PlayfairDisplay";
+  font-size: 0.93rem;
+  font-weight: normal;
+}
+
+.small-text--font {
+  font-family: Ubuntu;
+  font-size: 0.45rem;
+  font-weight: 300;
+}
+
+.content-text--font {
+  font-family: "PlayfairDisplay";
+  font-size: 0.45rem;
+  font-weight: normal;
+}
+
+/* Container */
+.container {
+  padding: 0 70px;
+}
+
+/* Navbar */
+.navbar {
+  display: flex;
+  align-items: center;
+  margin-top: 35px;
```

Comment:
why not here use margin from banner which is the same and have property like `margin: 35px 0;`